internal/provider/resource_gitlab_project_approval_rule_test.go

```diff
@@ -57,12 +58,82 @@ func TestAccGitLabProjectApprovalRule_basic(t *testing.T) {
 			},
 			// Update rule
 			{
-				Config: testAccGitlabProjectApprovalRuleConfig(project.ID, 2, projectUsers[1].ID, groups[1].ID, branches[1].ID),
+				Config: testAccGitlabProjectApprovalRuleConfig(project.ID, 2, "regular", projectUsers[1].ID, groups[1].ID, branches[1].ID),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckGitlabProjectApprovalRuleExists("gitlab_project_approval_rule.foo", &projectApprovalRule),
 					testAccCheckGitlabProjectApprovalRuleAttributes(&projectApprovalRule, &testAccGitlabProjectApprovalRuleExpectedAttributes{
 						Name:                "foo",
 						ApprovalsRequired:   2,
+						RuleType:            "regular",
+						EligibleApproverIDs: []int{currentUser.ID, projectUsers[1].ID, group1Users[0].ID},
+						GroupIDs:            []int{groups[1].ID},
+						ProtectedBranchIDs:  []int{branches[1].ID},
+					}),
+				),
+			},
+			// Verify import
+			{
+				ResourceName:      "gitlab_project_approval_rule.foo",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccGitLabProjectApprovalRule_any_approver(t *testing.T) {
```

Comment:
NIT:

```suggestion
func TestAccGitLabProjectApprovalRule_anyApprover(t *testing.T) {
```

Reply:
Actually, shouldn't it be `_AnyApprover` ?